src/test/com/constructions/Calculator_if_else.java

```diff
@@ -0,0 +1,27 @@
+package test.com.constructions;
+
+import java.util.Scanner;
+
+/**
+ * Умовна конструкція if - else.
+ * Створити клас Calculator і в ньому створити метод public void calculate (int number),
+ * який в залежності від того парна чи не парна цифра передана в нього буде виводити різні повідомлення у консоль.
+ */
+
+public class Calculator_if_else {
```

Comment:
ClassNameShouldBeInCamelCase. You don't use in Java "_" for naming excluding the special cases that you should really be sure to be valid.